src/main/java/green/earthquake/EarthquakeFrame.java

```diff
@@ -0,0 +1,113 @@
+        rbGroup.add(rbOne);
+        rbGroup.add(rbThirty);
+
+        rbOne.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                Disposable disposable = service.oneHour()
+                        // tells Rx to request the data on a background Thread
+                        .subscribeOn(Schedulers.io())
+                        // tells Rx to handle the response on Swing's main Thread
+                        .observeOn(SwingSchedulers.edt())
+                        //.observeOn(AndroidSchedulers.mainThread()) // Instead use this on Android only
+                        .subscribe(
+                                (response) -> handleResponse(response),
+                                Throwable::printStackTrace);
+            }
+        });
+
+        rbThirty.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                Disposable disposable = service.thirtyDays()
+                        // tells Rx to request the data on a background Thread
+                        .subscribeOn(Schedulers.io())
+                        // tells Rx to handle the response on Swing's main Thread
+                        .observeOn(SwingSchedulers.edt())
+                        //.observeOn(AndroidSchedulers.mainThread()) // Instead use this on Android only
+                        .subscribe(
+                                (response) -> handleResponse(response),
+                                Throwable::printStackTrace);
+            }
+        });
+
+
+        JPanel rbPanel = new JPanel(new GridLayout(0, 2));
+        rbPanel.add(rbOne);
+        rbPanel.add(rbThirty);
+
+        ListSelectionModel lsm = jlist.getSelectionModel();
+        lsm.addListSelectionListener(new ListSelectionListener() {
+            @Override
+            public void valueChanged(ListSelectionEvent e) {
+                int index = jlist.getSelectedIndex();
+                double lat = geo.get(index).getX();
+                double lon = geo.get(index).getY();
+                try {
+                    Desktop.getDesktop().browse(new URI(String.format("http://maps.google.com/?q=%f,%f", lat, lon)));
+                } catch (IOException | URISyntaxException ex) {
+                    throw new RuntimeException(ex);
+                }
+            }
+        });
+
+
+        add(rbPanel, BorderLayout.PAGE_START);
+        add(jlist, BorderLayout.CENTER);
+
+    }
+
+    private void handleResponse(FeatureCollection response) {
+
+        String[] listData = new String[response.features.length];
+        for (int i = 0; i < response.features.length; i++) {
+            Feature feature = response.features[i];
+            listData[i] = feature.properties.getMag() + " " + feature.properties.getPlace();
+            geo.add(new Point2D.Double(feature.geometry.getLatitude(), feature.geometry.getLongitude()));
```

Comment:
You can save the FeatureCollection so you don't need to save the geometry specifically